README.md

```diff
@@ -1,6 +1,130 @@
+which is essentially an implementation detail.
+
+## Multi-Namespace Quota 
+
+The AAQ Operator includes a Multi-Namespace Quota called ApplicationAwareClusterResourceQuota, 
+similar to the ClusterResourceQuota available in OpenShift distributions. 
+This quota is API compatible with ClusterResourceQuota, meaning it shares the same [specification and status definitions](https://github.com/kubevirt/application-aware-quota-api/blob/ac6757c09496bfab205d499a41daa6ed8815eab6/pkg/apis/core/v1alpha1/types.go#L257). 
+
+For more information, please see this [link](https://docs.openshift.com/container-platform/4.15/applications/quotas/quotas-setting-across-multiple-projects.html).
+
+Please see `manifests/examples/acrq-example.yaml` for an example manifest.
+
+## Pluggable policies for custom counting
+
+To define how you want resources to be counted for your application, you should set a sidecar container 
+in the AAQ object, where you configure the AAQ Operator.More details can be found in the "Configure AAQ Operator" section.
+
+To simplify the implementation of the sidecar container, you can use the [libsidecar](https://github.com/kubevirt/application-aware-quota-api/blob/main/libsidecar/sidecar.go) library. 
+To implement the sidecar container, you need to implement the following interface:
+
+```
+type SidecarCalculator interface {
+PodUsageFunc(podToEvaluate *corev1.Pod, existingPods []*corev1.Pod) (corev1.ResourceList, bool, error)
+}
+```
+
+The PodUsageFunc function takes as input the pod that needs to be evaluated and all the other pods in the cluster. 
+It returns the resources that should be counted for this pod, a boolean indicating if the pod belongs to an 
+application policy, and an error if something goes wrong during the evaluation.
+
+If a pod does not belong to any policy defined in the AAQ object, the native Quota evaluation will apply, 
+and the pod counting will be the same as the native ResourceQuota counting.
+
+If a pod belongs to multiple policies, the evaluation of all these policies will 
+apply to this pod, and the counting will be the sum of all policy evaluations.
+
+
+Please see `example_sidecars/label-sidecar/cmd/label-sidecar.go` for an example sidecar policy.
+
+
+### Configure AAQ Operator
+
+To configure AAQ Operator, set the following fields of the aaq object:
+
+* `aaq.spec.namespaceSelector` - determines in which namespaces AAQ's scheduling gate will be added to pods at creation time. This field follows the standard Kubernetes selector format.
+
+    ***Important Note***: if a namespace selector is not being defined, AAQ would target namespaces with the `application-aware-quota/enable-gating` label as default.
+
+    ***Warning***: Pods that set `.spec.nodeName` are not allowed to use in any namespace that AAQ is targeting therefore users shouldn't be targeting system namespaces where `.spec.nodeName` might be set.
```

Comment:
```suggestion
    ***Warning***: Pods that set `.spec.nodeName` are not allowed to use in any namespace that AAQ is targeting. Such pods would fail at creation time. Therefore, users shouldn't be targeting system namespaces where `.spec.nodeName` might be set.
```